### PR TITLE
save employee mappings to new employee_mappings table

### PR DESCRIPTION
**Pull request by Sravanksk**

### CODE REVIEW

**Comment by ashwin1111** [comment]
Please assign back review once you fix the form validators for the category mapping page

**Comment by ashwin1111** [comment]
https://fylein.slack.com/archives/C027VKVFSMN/p1660227635162649?thread_ts=1660227517.831859&cid=C027VKVFSMN

File: src/app/si/settings/employee-mappings/employee-mappings.component.ts [hunk 1]
```diff
@@ -75,44 +68,37 @@ export class EmployeeMappingsComponent implements OnInit {
...
-    const mappings = [];
-
-    employeeEVMappings.forEach(employeeEVMapping => {
-      mappings.push({
-        fyle_value: employeeEVMapping.source.value,
-        si_value: employeeEVMapping.destination.value,
-        ccc_value: that.getCCCAccount(that.employeeMappings, employeeEVMapping),
-        auto_mapped: employeeEVMapping.source.auto_mapped
-      });
+    that.employeeMappings = that.employeeMappings.filter((employeeMapping: EmployeeMapping) => {
```
Comment (ashwin1111) [inline src/app/si/settings/employee-mappings/employee-mappings.component.ts:71]: Why is this filter needed? Forgot the reason

File: src/app/si/settings/employee-mappings/employee-mappings.component.ts [hunk 2]
```diff
@@ -38,32 +40,23 @@ export class EmployeeMappingsComponent implements OnInit {
...
-      that.mappingsService.getMappings('EMPLOYEE', null, pageSize, 0, tableDimension).subscribe((employees) => {
-        that.count = that.columnsToDisplay.includes('ccc') ? employees.count / 2 : employees.count;
-        that.employeeMappings = employees.results;
-        that.isLoading = false;
-        const onboarded = that.storageService.get('onboarded');
-
-        if (onboarded) {
-          that.createEmployeeMappingsRows();
-        } else {
-          that.router.navigateByUrl(`workspaces/${that.workspaceId}/dashboard`);
```
Comment (ashwin1111) [inline src/app/si/settings/employee-mappings/employee-mappings.component.ts:64]: what happens to redirection? I don't find redirects in reset()

Comment (Sravanksk) [reply src/app/si/settings/employee-mappings/employee-mappings.component.ts:64]: fixed

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**